### DISCUSSION TITLE
Microsoft OAuth2 SMTP issues

### DISCUSSION THREAD

**Discussion by decomplexity** [PHPMailer/PHPMailer#2845] [General]
I have added to the Wiki (on 17th Dec 22) some  detailed notes of how SMTP is managed in AAD and EXO, with emphasis on scope operands that seem - from comments here and on Stackoverflow - to be the most common source of "535 5.7.3 Authentication unsuccessful" failures.

**Comment by Synchro** [comment]
Thanks
